test/Oscoin/Test/Crypto/Blockchain/Block/Helpers.hs

```diff
@@ -1,8 +0,0 @@
-module Oscoin.Test.Crypto.Blockchain.Block.Helpers where
-
-import           Oscoin.Crypto.Blockchain.Block (Beneficiary)
-import           Oscoin.Crypto.Hash (HasHashing, zeroShortHash)
-
--- | The default beneficiary.
-defaultBeneficiary :: HasHashing c => Beneficiary c
-defaultBeneficiary = zeroShortHash
```

Comment:
I've met once a colleague which was strongly against helpers module as in his eyes they were some sort of anti pattern -- perhaps he was right, who knows 😉